src/components/common/draftMeasureDialog/DraftMeasureDialog.tsx

```diff
@@ -117,7 +117,7 @@ const DraftMeasureDialog = ({ open, onClose, onSubmit, measure, loading }) => {
           />
         </Box>
         <>
-          {featureFlags?.qiCore6 && !measure?.model.includes("QDM") ? (
```

Comment:
IDE search gives me 14 matches for this flag. I think there are more files that needs to be updated.

Reply:
I'm getting a check for isQICore6, but none for the flag allowing qiCore to appear

Reply:
sorry, you are right.